ArjunaCore/arjuna/classes/com/arjuna/ats/internal/arjuna/objectstore/jdbc/drivers/edb_driver.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * JBoss, Home of Professional Open Source
+ * Copyright 2009, Red Hat Middleware LLC, and individual contributors
+ * as indicated by the @author tags.
+ * See the copyright.txt in the distribution for a
+ * full listing of individual contributors.
+ * This copyrighted material is made available to anyone wishing to use,
+ * modify, copy, or redistribute it subject to the terms and conditions
+ * of the GNU Lesser General Public License, v. 2.1.
+ * This program is distributed in the hope that it will be useful, but WITHOUT A
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+ * PARTICULAR PURPOSE.  See the GNU Lesser General Public License for more details.
+ * You should have received a copy of the GNU Lesser General Public License,
+ * v.2.1 along with this distribution; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,
+ * MA  02110-1301, USA.
+ *
+ * (C) 2009,
+ * @author JBoss by Red Hat.
+ */
+package com.arjuna.ats.internal.arjuna.objectstore.jdbc.drivers;
+
+
+/**
+ * JDBC store implementation driver-specific code. This version for PostgreSQL
```

Comment:
Can you please update the javadoc comment here to talk about PostgresPlus and not about PostgreSQL?

Reply:
It might be worth adding the version number too as it seems to change the meta data a bit too

Reply:
Good point. It also needs to extends _postgres_driver_ class instead of postgresql_driver.